Update concept section (#5834)
rerun-io/rerun: docs/content/concepts/blueprint.md

some exceptions to this rule at the moment, the intent is to eventually migrate
all state to the blueprint.)

## Current, default, and heuristics blueprints

> Blueprints may originate from multiple source.

Reviewer: ```suggestion
Blueprints may originate from multiple sources.
```

- The "current blueprint" for a given application ID is the one that is used by the viewer to display data at any given time. It is updated for each change made to the visualization within the viewer, and may be saved to a blueprint file at any time.
- The "default blueprint" is a snapshot that is set or updated when a blueprint is received from code or loaded from a file. The current blueprint may be reset to default blueprint at any time by using the "reset" button in the blueprint panel's header.

> - The "heuristic blueprint" is an automatically-produced blueprint based on the recording data. When no default blueprint is available, the heuristic blueprint is used when resetting the current blueprint. It is also possible to reset to the heuristic blueprint from the selection panel when an application ID is selected.

Reviewer: ```suggestion
- The "heuristic blueprint" is an automatically-produced blueprint based on the recording data. When no default blueprint is available, the heuristic blueprint is used when resetting the current blueprint. It is also possible to reset to the heuristic blueprint in the selection panel after selecting an application.
```
